cli/cmd/root.go

```diff
@@ -175,20 +177,45 @@ func updateRootUsage() {
 	})
 }
 
-func wasEnvFlagProvided() bool {
+func getCmdShorthandFlags(cmd *cobra.Command) strset.Set {
```

Comment:
I wonder if we could remove this function `getCmdShorthandFlags` and instead just implement `wasEnvFlagProvided` slightly to do the same thing. `getCmdShorthandFlags` only appears to be used in `wasEnvFlagProvided`.